torch/_torch_docs.py

```diff
@@ -11495,6 +11498,8 @@ def merge_dicts(*dicts):
            smallest elements
     sorted (bool, optional): controls whether to return the elements
            in sorted order
+    stable (bool, optional): makes the sorting routine stable, which guarantees that the order
+       of equivalent elements is preserved.
 
```

Comment:
We should probably mention the CPU-only support and error on CUDA inputs, and add tests for such cases...

Reply:
Thank you @nikitaved for the review. 

> We should probably mention the CPU-only support and error on CUDA inputs

Have updated the doc to mention this.